src/styles/main.scss

```diff
@@ -0,0 +1,58 @@
+.conteiner {
+  display: flex;
+  justify-content: center;
+  margin: 100px;
+}
+
+.arrow-minutes {
+  background-color: red;
+  width: 10px;
+  height: 150px;
+  border-top-left-radius: 5px;
+  border-top-right-radius: 5px;
+  margin-top: 50px;
+  margin-left: -11px;
+  transform-origin: bottom;
+  animation: time 3s linear alternate forwards;
+}
+
+@keyframes time {
+  from {
+    left: 0;
+  }
+
+  to {
+    transform: rotate(60deg);
+  }
+}
+
+.arrow-second {
+
+  background-color: royalblue;
+  width: 10px;
+  height: 200px;
+  border-top-left-radius: 5px;
+  border-top-right-radius: 5px;
+  transform-origin: bottom;
+  animation: second 3s linear  forwards;
+}
+
+@keyframes second {
```

Comment:
This animation you don't need. You should use the same animation for both hands.